src/views/ProductView.vue

```diff
@@ -1,21 +1,87 @@
 <template>
     <div class="d-flex flex-column ga-4">
         <div>
-            <v-toolbar class="bg-grey-lighten-4" density="compact">
+            <v-toolbar class="bg-grey-lighten-3" density="comfortable">
                 <v-btn icon="mdi-arrow-left" @click="router.back()"></v-btn>
-                <v-spacer></v-spacer>
             </v-toolbar>
+            <v-card class="bg-grey-lighten-3" rounded="0" variant="flat">
+                <v-card-item>
+                    <v-card-title class="mr-1">
+                        Product: {{ product?.product.internalName }}
+                    </v-card-title>
+
+                    <v-card-subtitle
+                        >ID: {{ product?.product.id }}</v-card-subtitle
+                    >
+                </v-card-item>
+                <v-card-text>
+                    <v-switch
+                        color="warning"
+                        density="comfortable"
+                        hide-details
+                        readonly
+                        :label="
+                            productIsHidden
+                                ? 'The product is hidden from customers.'
+                                : 'The product is publicly visible to customers.'
+                        "
+                        v-model="productIsHidden"
+                    ></v-switch>
+                    <p>
+                        Number of variants:
+                        {{ product?.product.variants.totalCount }}
+                    </p>
+                </v-card-text>
+            </v-card>
             <v-card class="bg-grey-lighten-4" rounded="0" variant="flat">
                 <v-card-item>
                     <v-card-title
-                        >Product Variant Version
+                        >Variant:
```

Comment:
Same with these examples here.
However is this is the only working version feel free to ignore these comments